src/test/java/com/iexec/worker/pubsub/StompClientTests.java

```diff
@@ -17,9 +18,10 @@
 import java.util.concurrent.TimeUnit;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.Assert.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.mockito.Mockito.*;
 
+@Tag("slow")
```

Comment:
Just an idea, what do you think about:
1. `void listenToSessionRequests(int sessionRefreshDelayMs) {` so we can drop slow tag
or
2. Use reflection so we can reset `SESSION_REFRESH_DELAY` to a small time unit
?

Reply:
How about [this](https://github.com/iExecBlockchainComputing/iexec-worker/pull/412/commits/c5afda7331b0c82af7c01741545061f75ce25f40)?

Reply:
Perfect :clap: :clap: Thanks!